mmv1/products/monitoring/AlertPolicy.yaml

```diff
@@ -966,20 +966,25 @@ properties:
     properties:
       - !ruby/object:Api::Type::String
         name: content
-        at_least_one_of:
-          - documentation.0.content
-          - documentation.0.mime_type
+        required: true
```

Comment:
Even if this is a valid change, we cannot increasing requirements for an existing field is a breaking change. Breaking changes are only permitted in major releases (5.0.0 was recent). Can you revert the requirement changes and file [a new issue](https://github.com/hashicorp/terraform-provider-google/issues/new/choose) to track changing this in a future release?

Reply:
Also, there are some merge conflicts that have popped up against the main branch. You will have to rebase/merge on the main branch

Reply:
There's a conflict because someone else got there first! #9360 Never mind, I'll close this now.